PSKoans/Koans/IntroTraining/AboutCmdletVerbs.koans.ps1

```diff
@@ -0,0 +1,214 @@
+
+            If you followed the example in the 'new' test, you can use Add-Content to add some
+            text to your newly created text file:
+            
+                $Path = "YOUR PATH.txt"
+                Add-Content -Path $Path -Value "Sgt. Bash is the best house robot because... fire."
+            
+            Before continuing, run this command several times. See what happens; is it the
+            result you expected?
+
+            You'll see that several lines of text were added to the file. This cmdlet only appends
+            information, it does not overwrite. 
+
+            Basically, if it doesn't exist then a cmdlet with the add verb can probably be used
+            to make it so.
+
+            A common example an Operations Engineers may see is with Office 365 calendar permissions. 
+            If an Engineer wants to grant permissions to somebody who doesn't have any, they use Add.
+            If they want to change permissions for somebody who already has some, they need to use a
+            different verb that we'll cover later.
+        #>
+
+        It 'is for commands that append data' {
+            <#
+                Using the Get-Command cmdlet, which retrieves all the cmdlets currently available,
+                find 5 new add commands you haven't previously used.
+            #>
+
+            # Replace each __ with the name of a Add cmdlet
+            $Answers = "____","____","____","____","____"
+            $Answers | Should -BeIn (Get-Command -Verb Add).Name
+
+            $UniqueCheck = $Answers | Get-Unique
+            $UniqueCheck.count -eq 5 | 
+            Should -BeTrue -Because "five unique cmdlets should be supplied"
+
+        }
+    }
+
+    Context "Set" {
+        <#
+            Set
+
+            Cmdlets with the Set verb will overwrite information that already exists.  
+
+            If you followed the example in the 'New' and 'Add' tests, you can use set to do
+            something to your text file:
+            
+                $Path = "YOUR PATH.txt"
+                Set-Content -Path $ItemPath -Value "Sir Kill-A-Lot is the best house robot because of reasons."
+            
+            Before continuing, run this command several times. See what happens, is it the
+            result you expected?
+
+            You'll see that there's only one line of text in the file. This is because the Set-Content command
+            will overwrite information that's already there.
+
+            Some Set-* cmdlets require the instance to already be present for you to change it;
+            you'll need to use a New-* cmdlet first to create an instance before you can 
+            overwrite information within it.
+
+            A common example an Operations Engineers may see is with Office 365 calendar permissions. 
+            If somebody already has permissions an Engineer may use a Set cmdlet to change the user's
```

Comment:
```suggestion
            If a user already has some permissions configured, you can use a Set cmdlet to change them.
```
And again, you could potentially use the actual cmdlet's name. 🙂

Reply:
🦄